frontend/workflows-lib/lib/components/workflow/TasksTable.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Box, Paper, Typography, useTheme } from "@mui/material";
+import { Task } from "../../types";
+import { getStatusIcon } from "../common/StatusIcons";
+
+interface TaskTableProps {
+  tasks: Task[];
+}
+
+const TasksTable: React.FC<TaskTableProps> = ({ tasks }) => {
+  const theme = useTheme();
+  return (
+    <Box display="flex" flexWrap="wrap" gap={1.5} padding={2} overflow="auto">
+      {tasks.map((task) => (
+        <Paper
+          elevation={8}
+          sx={{
+            padding: theme.spacing(1.5),
+            minWidth: 180,
+            maxWidth: 180,
+          }}
+        >
+          <Box
+            display="flex"
+            justifyContent="space-between"
+            alignItems="center"
+            minWidth={120}
+          >
```

Comment:
Do we need a `box` inside the `paper` or could we style the paper to make it work without?